app/src/main/res/values/colors.xml

```diff
@@ -65,4 +65,5 @@
     <color name="theme_background">@android:color/white</color>
     <color name="icon_background">@android:color/white</color>
     <color name="logo_background">@color/blue_black</color>
+    <color name="qr_background">#DDFFFFFF</color>
```

Comment:
Please!!! use more descriptive key. `qr` is very generic prefix. i can't get context of usage of key.